test/reconciler-tests.sh

```diff
@@ -27,6 +27,8 @@ if [ "${EVENTING_KAFKA_BROKER_CHANNEL_AUTH_SCENARIO:-""}" != "" ]; then
   success
 fi
 
+export BROKER_CLASS="Kafka"
```

Comment:
will we run them eventually w/ `NamespacedBrokerClass`  / `KafkaNamespaced` too?

Reply:
Yes, I will. I will do that in a separate PR.

That actually needs some prow job changes too